README.md

```diff
@@ -1 +1,23 @@
-# Microverse-portfolio
\ No newline at end of file
+# Setup-and-mobile-version-skeleton
+
+> This is my portfolio Microverse project
+Additional description about the project and its features.
+
+## Built With
+
+- html
+- css
+- github
+
+## Author
+👤 **Author1**
+
+- GitHub: [dadadon](https://github.com/dadadon)
+- Twitter: [Davian Beroni](https://twitter.com/davianberoni)
+- LinkedIn: [Davian Beroni](https://www.linkedin.com/in/davian-beroni-502351b7/)
+
+## Acknowledgments
+
+- Microverse
+
+## 📝 License This project is MIT licensed. 
```

Comment:
- Please, kindly update the MIT license link. You can use the MIT from the [microverse readme template](https://github.com/microverseinc/readme-template/blob/master/README.md) and add it to the root directory of your project to make your readme appear more professional.

Reply:
*.gitignore*

- Please, kindly add the `node_modules/` directory in the `.gitignore` file to prevent pushing its dependencies into your repo.